### PR TITLE
Node 12 in CI

### PR DESCRIPTION
**Pull request by xtuc**
**What kind of change does this PR introduce?**

**Did you add tests for your changes?**

**Does this PR introduce a breaking change?**

**What needs to be documented once your changes are merged?**

We want to test against `WebAssembly.Global` which is available in Node 11.

This PR allows webpack to be tested against arbitrary Node nightly.

### CODE REVIEW

**Comment by xtuc** [comment]
Some stat tests are failing, we should probably fix them first. I don't get how `allow_failures` works, I tried to match on the env, see https://github.com/webpack/webpack/pull/8182/commits/a7159c96ba10f8b9d26de3add3b15eab9cf539a1.

**Comment by sendilkumarn** [comment]
@xtuc Can we just configure the CI to run `wasm` related tests in `nodejs11`? That might reduce the surface area of random failures.

**Comment by xtuc** [comment]
@sendilkumarn good point! What should I pass, something like `jest -t wasm` right?

**Comment by sendilkumarn** [comment]
yeah something similar to that

File: .travis.yml [hunk 1]
```diff
@@ -44,16 +44,35 @@ matrix:
...
 install:
-  - yarn --frozen-lockfile
-  - yarn link --frozen-lockfile || true
-  - yarn link webpack --frozen-lockfile
+  - yarn --frozen-lockfile $YARN_EXTRA_ARGS
+  - yarn link --frozen-lockfile $YARN_EXTRA_ARGS || true
+  - yarn link webpack --frozen-lockfile $YARN_EXTRA_ARGS
 
-script: yarn travis:$JOB_PART
+script: yarn travis:$JOB_PART $YARN_EXTRA_ARGS
```
Comment (sokra) [inline .travis.yml]: Is `$YARN_EXTRA_ARGS` really needed here?

**Comment by xtuc** [comment]
Interesting, this version of Node changed the OOM detection I guess. I can try to increase the memory a wasm module can allocate (I'm on the phone sorry).

**Comment by sokra** [comment]
I think it's a memory leak in the test suite. I try to address it in a separate PR.

**Comment by sokra** [comment]
Thanks :tada

**Pull request merged**